src/foreman.ts

```diff
@@ -26,6 +26,13 @@ async function getReleases(octokit: GitHub): Promise<GitHubRelease[]> {
   return releases;
 }
 
+function filterValidReleases(releases: GitHubRelease[]): GitHubRelease[] {
+  return releases.filter(release => {
+    const tag = release.tag_name;
+    return tag.startsWith("v") && semver.valid(tag);
```

Comment:
I assume that `semver.valid` must handle the leading `v` just fine, since your tests appear to pass